apps/gitness/src/pages/project-settings/project-settings-members-page.tsx

```diff
@@ -25,41 +27,36 @@ import { timeAgoFromEpochTime } from '../pipeline-edit/utils/time-utils'
 import { useQueryState, parseAsInteger } from 'nuqs'
 import { PageResponseHeader } from '../../types'
 import { DialogState } from './types'
+import { useDebouncedQueryState } from '../../hooks/useQuery'
 
-const filterOptions = [{ name: 'Filter option 1' }, { name: 'Filter option 2' }, { name: 'Filter option 3' }]
 const SortOptions = [
   { name: 'Name', value: 'name' },
   { name: 'Created', value: 'created' }
 ]
 
 const ProjectSettingsMemebersPage = () => {
   const space_ref = useGetSpaceURLParam()
-
   const [dialogState, setDialogState] = useState<DialogState>({
     isDialogDeleteOpen: false,
     selectedMember: null
   })
-
-  const { sort, query: currentQuery } = useCommonFilter<MembershipListQueryQueryParams['sort']>()
-  const [query, setQuery] = useQueryState('query', {
-    defaultValue: currentQuery || ''
-  })
+  const { sort } = useCommonFilter<MembershipListQueryQueryParams['sort']>()
```

Comment:
we're still using `useCommonFilter`? Will this be removed in a separate PR?

Reply:
This is still used for `sort` in few pages. Will remove it separately.